apps/oxlint/vitest.config.ts

```diff
@@ -2,6 +2,7 @@ import { configDefaults, defineConfig } from 'vitest/config';
 
 export default defineConfig({
   test: {
+    setupFiles: ['./vitest_setup.ts'],
     exclude: [...configDefaults.exclude, 'fixtures/**'],
   },
```

Comment:
```suggestion
  test: {
    exclude: [...configDefaults.exclude, 'fixtures/**'],
  },
  define: {
    DEBUG: 'true'
  }
```

We can use vite's built in define plugin rather than a manual implementation